dockerUtils/go.mod

```diff
@@ -1,24 +1,36 @@
 module github.com/nestybox/sysbox-libs/dockerUtils
 
-go 1.14
+go 1.18
+
+require (
```

Comment:
why two "require" sections?

Reply:
`go mod tidy` automatically generates those two sections now, one for direct dependencies and one for indirect.